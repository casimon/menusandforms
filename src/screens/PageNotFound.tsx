import React from 'react'

import { makeStyles } from '@material-ui/core/styles'
import Grid from '@material-ui/core/Grid'
import Typography from '@material-ui/core/Typography'
import Container from '@material-ui/core/Container'

const useStyles = makeStyles((theme) => ({
  root: {
    flexGrow: 1,
    padding: theme.spacing(2),
  },
}))

function PageNotFound() {
  const classes = useStyles()

  return (
    <div className={classes.root}>
      <Container maxWidth='sm'>
        <Grid
          container
          direction='column'
          justify='center'
          alignItems='stretch'
          spacing={3}
        >
          <Grid item xs={12}>
            <Typography variant='h5' component='h1' gutterBottom>
              Page not implemented
            </Typography>
          </Grid>
        </Grid>
      </Container>
    </div>
  )
}

export default PageNotFound
