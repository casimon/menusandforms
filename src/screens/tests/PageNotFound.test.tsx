import React from 'react'

import { render } from '@testing-library/react'
import PageNotFound from '../PageNotFound'

const init = () => render(<PageNotFound />)

describe('My Page Not Found component', () => {
  test('should render', () => {
    const { container } = init()
    expect(container).toMatchSnapshot()
  })
})
