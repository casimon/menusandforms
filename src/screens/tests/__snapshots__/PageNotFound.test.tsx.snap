// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`My Page Not Found component should render 1`] = `
<div>
  <div
    class="makeStyles-root-1"
  >
    <div
      class="MuiContainer-root MuiContainer-maxWidthSm"
    >
      <div
        class="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-3 MuiGrid-direction-xs-column MuiGrid-justify-xs-center"
      >
        <div
          class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12"
        >
          <h1
            class="MuiTypography-root MuiTypography-h5 MuiTypography-gutterBottom"
          >
            Page not implemented
          </h1>
        </div>
      </div>
    </div>
  </div>
</div>
`;
